.github/CODEOWNERS

```diff
@@ -1,8 +1,14 @@
 # Documentation/Infrastructure
-/admin                                      @mukeshpanchal27 @felixarntz
-/tests/admin                                @mukeshpanchal27 @felixarntz
-/bin                                        @mukeshpanchal27 @felixarntz
 /.github                                    @mukeshpanchal27 @felixarntz @thelovekesh
+/bin                                        @mukeshpanchal27 @felixarntz @thelovekesh
```

Comment:
I added @thelovekesh to `bin` codeowners.

Reply:
Who?